### ISSUE TITLE
Issue with using Multiple Buttons while using Motion

### ISSUE THREAD

**Issue by saieno** [r57zone/DualShock4-emulator#78]
I absolutely love the DualShock4-Emulator and its working great in RPCS3. Was able to use it for HAZE to shake off being on fire.

One issue I'm running into however is with Folklore. In Folklore you need to hold the Right Shoulder button, then pull the controller UpWards to catch a creature. By default the Right Shoulder button is the Shake button, so I remapped it in the config. However, whenever I'm holding the Right Shoulder button to initiate catching a creature, then use the Back Botton to trigger the motion, it lets go of the Right Shoulder button.

It seems like I can't have a different button pressed independantly of triggering the motion. Any suggestions or work arounds?

**Comment by KelseyHigham** [comment]
this also affects Noby Noby Boy, which requires you to hold R1 and tilt the controller to zoom the camera.

this happens during the tutorial, so the game is unplayable.

**Comment by r57zone** [comment]
Can also bind MULTI KEY to a button other than back.

You can also try motion emulation using smartphone if it supports a gyroscope.

**Comment by saieno** [comment]
> Can also bind MULTI KEY to a button other than back.
> 

The problem is, when the motion is triggered, no other buttons work. For example, if BACK is held to trigger motion, and I need to hold R1 as well for a game motion, the button stops pressing and the motion takes over. Even if I need to hold multiple buttons while ago doing motion, that's not possible with this solution.

**Comment by KelseyHigham** [comment]
i'd like to hold the MultiActivateKey and press non-chorded buttons at the same time.

for example, if my config looks like this:

```
MultiActivateKey=BACK
MotionShakingKey=NONE
MotionUp=DPAD-UP
MotionDown=DPAD-DOWN
MotionLeft=DPAD-LEFT
MotionRight=DPAD-RIGHT
```

then i'd like to be able to hold `"BACK" + "RIGHT-SHOULDER" + "DPAD-LEFT"`, in order to send `"RIGHT-SHOULDER" + "MotionLeft"` to the game.

**Comment by r57zone** [comment]
I have enabled duplicate motion buttons presses in the [new version](https://github.com/r57zone/DualShock4-emulator/releases).

**Comment by saieno** [comment]
> I have enabled duplicate motion buttons presses in the [new version](https://github.com/r57zone/DualShock4-emulator/releases).

The issue I'm having is I need to be able to hold down a button in-game, which isn't assigned to any of the DS4Emulator buttons, and then trigger motion _without the held button being interrupted_. For example, if I hold down R1 in-game, I want that to continue to be held down in-game even if I trigger motion by pushing Back and using the Control Sticks or D-Pad. 

A few examples requiring this are capturing an enemy in Folklore, or turning the valves in Killzone 2. For example, here's how the motion control requirements for Killzone 2 function: [https://www.youtube.com/watch?v=QL63SV2ncfw](https://www.youtube.com/watch?v=QL63SV2ncfw ) and without being able to hold the triggers down while also doing motion, its not possible to complete.

**Comment by r57zone** [comment]
@saieno Does this still not work in the new version?

**Comment by saieno** [comment]
> @saieno Does this still not work in the new version?

That's correct. In the new version I don't really notice a functional difference with how I was using it previously.
